cypress/integration/movie_list_spec.js

```diff
@@ -0,0 +1,119 @@
+describe('Movie List', function() {
+  beforeEach(function() {
+    cy.visit('/')
+  })
+
+  context('12 Angry Men', function() {
+    beforeEach(function() {
+      cy.get('#movie-list .movie-row:first').as('firstMovie')
+    })
+
+    it('should be first', function() {
+      cy.get('@firstMovie').contains('12 Angry Men')
```

Comment:
There are some assumptions made here that `12 Angry Men` will always be the first movie.  The app has static data (no XHR, oddly enough), but if it _were_ dynamic then either `fixtures/movies.json` would exist or XHR would be mocked to ensure a idempotent/deterministic payload.